mapie/estimator/estimator.py

```diff
@@ -278,10 +279,10 @@ def _aggregate_with_mask(
         ArrayLike of shape (n_samples_test,)
             Array of aggregated predictions for each testing sample.
         """
-        if self.method in self.no_agg_methods_ or self.cv in self.no_agg_cv_:
+        if self.method in self.no_agg_methods_ or self.use_split_method:
             raise ValueError(
                 "There should not be aggregation of predictions "
-                f"if cv is in '{self.no_agg_cv_}' "
+                f"if cv is in '{self.no_agg_cv_}', if cv >=2 "
```

Comment:
Why do we return `self.no_agg_cv_` in the error message when we use `self.use_split_method` in the if condition?

Reply:
Because `self.use_split_method` is computed based on `self.no_agg_cv_` in other part of the code.